### ISSUE TITLE
backslash-escaped newlines after non-literals break parser

### ISSUE THREAD

**Issue by ktkopone** [ikamensh/flynt#68]
minimal example:
```
#!/usr/bin/env python3

blah1 = 'a'
blah2 = blah1 \
        + 'b'
print(blah2)
```

The line with the backslash will cause the following error:
```
λ flynt blah.py --transform-concats -v
Running flynt v.0.48.1
Skipping fstrings transform of file <...>/blah.py due to
Traceback (most recent call last):
  File "c:\program files\python38\lib\site-packages\flynt\api.py", line 43, in _fstringify_file
    new_code, concat_changes = fstringify_concats(
  File "c:\program files\python38\lib\site-packages\flynt\process.py", line 146, in fstringify_concats
    return _transform_code(
  File "c:\program files\python38\lib\site-packages\flynt\process.py", line 158, in _transform_code
    return jt.fstringify_code_by_line()
  File "c:\program files\python38\lib\site-packages\flynt\process.py", line 49, in fstringify_code_by_line
    self.try_chunk(chunk)
  File "c:\program files\python38\lib\site-packages\flynt\process.py", line 104, in try_chunk
    self.maybe_replace(chunk, contract_lines, converted, rest)
  File "c:\program files\python38\lib\site-packages\flynt\process.py", line 108, in maybe_replace
    if get_quote_type(str(chunk)) in (qt.triple_double, qt.triple_single):
  File "c:\program files\python38\lib\site-packages\flynt\format.py", line 27, in get_quote_type
    return token.get_quote_type()
  File "c:\program files\python38\lib\site-packages\flynt\lexer\PyToken.py", line 46, in get_quote_type
    assert self.toknum is token.STRING
AssertionError
fstringifying <...>\blah.py...no change

Flynt run has finished. Stats:

Execution time:                            0.007s
Files modified:                            0
```

Note that if the concat line's `blah1` is replaced with its actual raw value `'a'`, the error will not occur.

Also as a second minor bug note that when not in verbose mode, the traceback will not be shown but the output will still prefix with:
`Skipping fstrings transform of file C:\Users\kok1wte\repos\ghsm\blah.py due to` (followed by nothing).

**Comment by ktkopone** [comment]
Same issue seems to occur on multiline concats that are enclosed in parentheses:
```
#!/usr/bin/env python3

blah1 = 'a'
blah2 = (blah1
         + 'b')
print(blah2)
```

**Comment by ikamensh** [comment]
Hi @ktkopone , thanks for the bug report.

Fixed the main bug in v0.49 (to be released soon). For  the minor bug, this is the root cause: https://stackoverflow.com/questions/5009238/assertionerror-returning-empty-string-in-python

I will modify code to work around this.
